CHANGES.md

```diff
@@ -9,6 +9,8 @@ To be released.
 ### Backward-incompatible API changes
 
  -  Removed `LiteDBStore` class.  Use `DefaultStore` instead.  [[#662]]
+ -  Removed `BlockChain<T>.Contains(TxId)` method. Use
```

Comment:
It would be better if we split lines right after the period and start a new line with the beginning of the next sentence.